### PR TITLE
3 ➡️ 4 

### PR DESCRIPTION
**Pull request by chapulina**
# ➡️ Forward port

Port `ign-common3` to `main`

Branch comparison: https://github.com/ignitionrobotics/ign-common/compare/main...ign-common3

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸🔸

### CODE REVIEW

File: tutorials/profiler.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ In order to use the profiler, inspection points must be added to the source code
 and the application or library must be linked to the `ignition-common::profiler`
 component.
 
-To start, download the [profiler.cc](https://github.com/ignitionrobotics/ign-common/raw/master/examples/profiler.cc) example.
+To start, download the [profiler.cc](https://github.com/ignitionrobotics/ign-common/raw/ign-common3/examples/profiler.cc) example.
```
Comment (ahcorde) [inline tutorials/profiler.md:28]:
```suggestion
To start, download the [profiler.cc](https://github.com/ignitionrobotics/ign-common/raw/main/examples/profiler.cc) example.
```

**Review by ahcorde — CHANGES_REQUESTED** [review]

File: tutorials/profiler.md (hunk 1 above)
Comment (chapulina) [reply tutorials/profiler.md]: Fixed

**Comment by chapulina** [comment]
I don't know why the actions CI is not being triggered. Closing and reopening to nudge it.

**Comment by chapulina** [comment]
It looks ok now :grimacing:

**Comment by adlarkin** [comment]
It looks like this forward port is missing commit https://github.com/ignitionrobotics/ign-common/commit/798f8384747975abb3c36d0ee5f831a747070f41. Can we add it in?

**Comment by mjcarroll** [comment]
> It looks like this forward port is missing commit 798f838. Can we add it in?

I would prefer to forward port this one separately, if that's okay.

**Comment by chapulina** [comment]
All :green_circle:

**Review by adlarkin — APPROVED** [review]

**Pull request merged**